run.sh

```diff
@@ -12,6 +12,7 @@ SENTRY_ORG=testorg-az
 SENTRY_PROJECT=prithvi-test-vue
 PREFIX=assets
 
+rm -rf dist
```

Comment:
I like to be explicit because I've seen cases where sometimes builds do not re-build, or some things are missed, depending what shell/environment you're running from.

I wasn't sure if it was rebuilding at one point, so I inserted this, for debugging. We also have this in our other run.sh deploy.sh's like in our React app.